README.md

```diff
@@ -23,6 +23,8 @@ Set 'Last Modified Solr Field' and 'Maximum number of Islandora links to process
 
 ![Configuration](https://camo.githubusercontent.com/407972e0a2c14bafd74924992c659021b800abb0/687474703a2f2f692e696d6775722e636f6d2f455a534f4b68372e706e67)
 
+**WARNING!** Larger sites with > 100,000 objects may want to uncheck the "Prefetch URL aliases during sitemap generation" option found on the xmlsitemap admin configuration page (/admin/config/search/xmlsitemap/settings) if they encounter issues during the sitemap indexing process.
```

Comment:
@DiegoPino Sure! I'll make a release pull as soon as I get confirmation that the actual body of the message is :+1: